requirements.txt

```diff
@@ -1,2 +1,2 @@
-pennylane>=0.22
+git+https://github.com/PennyLaneAI/pennylane
```

Comment:
Would we like to change to `pennylane>=0.28` in a separate PR or change here and merge this PR after releasing PennyLane?

Reply:
Actually I think we only need Pennylane=0.27. That's when the `decompose_hamiltonian` method was swapped.